apt_info.py

```diff
@@ -35,8 +35,12 @@ def _convert_candidates_to_upgrade_infos(candidates):
     changes_dict = collections.defaultdict(lambda: collections.defaultdict(int))
 
     for candidate in candidates:
+        # The 'now' archive only shows that packages are not installed. We tend
+        # to filter the candidates on those kinds of conditions before reaching
+        # here so here we don't want to include this information in order to
+        # reduce noise in the data.
         origins = sorted(
-            {f"{o.origin}:{o.codename}/{o.archive}" for o in candidate.origins}
+            {f"{o.origin}:{o.codename}/{o.archive}" for o in candidate.origins if o.archive !='now'}
```

Comment:
linter is unhappy with this one:

```suggestion
            {f"{o.origin}:{o.codename}/{o.archive}" for o in candidate.origins if o.archive != 'now'}
```

Reply:
also, is the count still correct with this change? I imagine it just removes the needless origin, but did you make sure the number of install packages metric total still correctly reflect the actual installed count?

Reply:
huh interesting, I just saw the failed flake8 test in CI. I'm using pywright locally because it was simpler to integrate into my editor + plugins but it's showing nothing for the problems that flake8 found. I guess that means it's time I made the effort of switching away from pywright.

It'll be fixed in my next push